pkg/controller/config/config_controller_test.go

```diff
@@ -319,7 +319,15 @@ func TestConfig_DeleteSyncResources(t *testing.T) {
 			},
 		},
 	}
+
 	ctx, cancelFunc := context.WithCancel(context.Background())
+	once := gosync.Once{}
+	defer func() {
```

Comment:
How could this defer be called multiple times, such that we need a `once.Do()`?

Reply:
n/a after latest commit